lib/http/chainable.rb

```diff
@@ -224,7 +231,10 @@ def default_options
     end
 
     # Set options for HTTP
-    # @param opts
+    #
+    # See HTTP::Options.defined_options for list of options.
```

Comment:
Please use `@see` tag.

``` ruby
# @see Options
```

Notice that `.defined_options` are implementation details and should not be referenced in documentation.